src/lib/theme/default.ts

```diff
@@ -630,7 +630,7 @@ const theme: FlowbiteTheme = {
         icon: 'h-5 w-5',
       },
       selector: {
-        base: 'w-12 border border-gray-300 bg-white py-2 leading-tight text-gray-500 hover:bg-gray-100 hover:text-gray-700 dark:border-gray-700 dark:bg-gray-800 dark:text-gray-400 dark:hover:bg-gray-700 dark:hover:text-white',
+        base: 'w-12 border border-gray-300 py-2 leading-tight text-gray-500 hover:bg-gray-100 hover:text-gray-700 dark:border-gray-700 dark:bg-gray-800 dark:text-gray-400 dark:hover:bg-gray-700 dark:hover:text-white',
```

Comment:
I think that removing the `bg-white` here isn't the best approach, 'cause, well... you will still have the problem with `dark:bg-gray-800`... Right? So, I would instead look at the `active` and make the `bg-blue-50` and `dark:bg-gray-700` important so that they can overwrite the standard bg from the base.

Reply:
@rluders totally makes sense. I will perform this little change.

Reply:
@rluders Done! Check it please as soon as you can :D